pages/workflow/review.mdx

```diff
@@ -0,0 +1,147 @@
+
+<video
+  controls
+  loop
+  muted
+  width="100%"
+  autoPlay
+  src={require('../../images/workflow/2.review/add-collaborators.mp4')}
+  alt="Add Collaborators"
+/>
+
+_Example: Add reviewer as branch collaborators_
+
+- Share a direct link to the branch with the reviewers: Go to the branch review screen and copy the browser URL.
+
+> 💡 Follow the same process for requesting branch reviews on Avo as you would for other types of internal requests. For example if your team mostly works in Asana or Jira, create an Asana task or Jira ticket with a link to the branch review screen.
+
+We also recommend you set up the Slack workflow integration for your Avo workspace. The Slack workflow integration will post all branch updates and comments to a Slack channel of your choice. 
+
+ <div className={styles.row}>
+          <PageLink
+            image={DevIcon}
+            title="Slack notifications"
+            description="Learn how to enable Slack notifications in your Avo Workspace"
+            callToAction={
+              new CallToAction('https://www.avo.app/docs/workspace/integrations#slack')
+            }
+          />
+  </div>
+
+Additionally, with approval workflows you can configure how many approvals are required, and who can approve, before the branch can be merged.
+
+ <div className={styles.row}>
+          <PageLink
+            image={AnalyticsManagerIcon}
+            title="Approval workflows"
+            description="Increase the quality of your tracking plan with protected main branch and branch approvals"
+            callToAction={
+              new CallToAction('https://www.avo.app/docs/workspace/approval-workflows')
+            }
+          />
+  </div>
+
+## Submit a review
+
+The role of the reviewer(s) is to make sure that the suggested changes fit your data needs, are according to the existing standards and naming conventions, and are feasible for implementation. 
+
+The branch review screen in Avo summarizes the changes that have been suggested on the branch. 
+
+On the review screen, you can comment on every item that has been introduced or changed on your branch, ask questions and suggest changes.
+
+<center>
+  <Zoom>
+    <img
+          width="100%"
+          src={require('../../images/workflow/2.review/add-comments.png')}
+          alt="Adding comment"
+    />
+  </Zoom>
+</center>
+
+_Example: Adding comment to the changes_
+
+To see more details about the items being changed you can click into each item on the review screen. You can add comments directly to each item. All comments made on the branch will show up on the review screen.
+
+<video
+  controls
+  loop
+  muted
+  width="100%"
+  autoPlay
+  src={require('../../images/workflow/2.review/commenting-in-an-item.mp4')}
```

Comment:
This mp4 is 1mb, can we compress the mp4s somehow before to reduce the file sizes people have to download while reading the docs?